src/containers/Movie/component.js

```diff
@@ -0,0 +1,39 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import { Row, Col, Spin } from 'antd'
+import classnames from 'classnames'
+
+import MovieImages from './MovieImages'
+import MovieInfo from './MovieInfo'
+import Credits from './Credits'
+
+const Movie = ({ movieId, loading }) => (
+  <>
+    {loading && (
+      <Row
+        className="top-margin"
+        justify="center"
+      >
+        <Col>
+          <Spin />
+        </Col>
+      </Row>
+    )}
+    <div className={classnames({ 'd-none': loading })}>
```

Comment:
why not to do conditional rendering instead?

Reply:
redone